### PR TITLE
[#7262] Remove shortcut creation when following universal links insid…

### LINKED ISSUE #7262: Remove shortcut creation when following universal links inside status
As a user, I don't want to have extra unusable items created in the home screen.
See more details [here](https://github.com/status-im/status-react/pull/7218#issuecomment-453102062)

### Description

*Type*: Bug

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #7262

### CODE REVIEW

File: src/status_im/browser/core.cljs [hunk 1]
```diff
@@ -239,24 +233,30 @@
...
-  (let [{:strs [url loading title]} (js->clj event)]
-    (fx/merge cofx
-              (update-browser-option :loading? loading)
-              (update-browser-name title)
-              (update-browser-on-nav-change url error?))))
+  (let [{:strs [url loading title]} (js->clj event)
+        deep-link? (utils.universal-links/deep-link? url)]
+    (if (utils.universal-links/universal-link? url)
+      (if (and deep-link? platform/ios?) ;; ios webview handles this
+        (fx/merge cofx {})
```
Comment (yenda) [inline src/status_im/browser/core.cljs]: seems like this does nothing? so you could replace the `if` by a `when-not` and remove that line

**Review by yenda — CHANGES_REQUESTED** [review]

File: src/status_im/utils/universal_links/core.cljs [hunk 2]
```diff
@@ -46,6 +46,10 @@
   (boolean
    (re-matches #"((^https?://get.status.im/)|(^status-im://))[\x00-\x7F]+$" url)))
 
+(defn deep-link? [url]
+  (boolean
+   (re-matches #"(^status-im://)[\x00-\x7F]+$" url)))
```
Comment (goranjovic) [inline src/status_im/utils/universal_links/core.cljs]: let's put those regex literals into constants.
also, can we have some unit tests for regexes?

**Review by goranjovic — CHANGES_REQUESTED** [review]

**Review by dmitryn — APPROVED** [review]
lgtm, except points already mentioned by other reviewers

**Comment by flexsurfer** [comment]
@yenda @goranjovic thanks, made changes!

**Review by yenda — APPROVED** [review]

**Review by goranjovic — APPROVED** [review]

**Comment by goranjovic** [comment]
@flexsurfer did you push the changes? I still see the same thing

**Comment by flexsurfer** [comment]
@goranjovic yes, i did, don't know what can be the reason, i can see changes

**Comment by churik** [comment]
Checked on IOS and Android: no shortcuts are created when following universal links on public and 1-1 chats in Status.
As a regression checked Mac OS build, and following universal links from browsers on IOS and Android.
Waiting for e2e results.

**Pull request merged**